amq/amq62-basic.json

```diff
@@ -18,62 +18,72 @@
         {
             "description": "The name for the application.",
             "name": "APPLICATION_NAME",
+            "displayName": "Application Name",
             "value": "broker",
             "required": true
         },
         {
             "description": "Protocols to configure, separated by commas. Allowed values are: `openwire`, `amqp`, `stomp` and `mqtt`.",
             "name": "MQ_PROTOCOL",
+            "displayName": "Protocols",
             "value": "openwire",
             "required": false
         },
         {
             "description": "Queue names, separated by commas. These queues will be automatically created when the broker starts. If left empty, queues will be still created dynamically.",
             "name": "MQ_QUEUES",
+            "displayName": "Queues",
             "value": "",
             "required": false
         },
         {
             "description": "Topic names, separated by commas. These topics will be automatically created when the broker starts. If left empty, topics will be still created dynamically.",
             "name": "MQ_TOPICS",
+            "displayName": "Topics",
             "value": "",
             "required": false
         },
         {
             "description": "List of packages that are allowed to be serialized for use in ObjectMessage, separated by commas. If your app doesn't use ObjectMessages, leave this blank. This is a security enforcement. For the rationale, see http://activemq.apache.org/objectmessage.html",
             "name": "MQ_SERIALIZABLE_PACKAGES",
+            "displayName": "Serializable Packages",
             "value": "",
             "required": false
         },
         {
             "description": "User name for standard broker user. It is required for connecting to the broker. If left empty, it will be generated.",
             "name": "MQ_USERNAME",
+            "displayName": "Username",
```

Comment:
Will it be clear what the username is for? Shouldn't it be "MQ Username" or something similar. I feel like just "Username" is not specific enough and may be confusing to the user.